src/Dynamo/src/HyparDynamo/Hypar.cs

```diff
@@ -2,55 +2,66 @@
 using System.Collections.Generic;
 using System.Linq;
 using Elements.Serialization.glTF;
-using Elements.Serialization.JSON;
 using Hypar.Revit;
 using RevitServices.Persistence;
 
-
 namespace HyparDynamo.Hypar
 {
     public static class Wall
     {
         /// <summary>
-        /// gets the walls
+        /// Convert Revit walls to Elements.WallsByProfiles for use in Hypar models
         /// </summary>
-        /// <param name="wall">The walls to be exported</param>
-        /// <returns name="Hypar.Wall">The Hypar Wall element </param>
-        public static Elements.Wall[] FromRevitWall( this Revit.Elements.Wall RevitWall) 
+        /// <param name="RevitWall">The walls to be exported</param>
+        /// <returns name="WallByProfile">The Hypar walls</param>
+        public static Elements.WallByProfile[] FromRevitWall(this Revit.Elements.Wall RevitWall)
         {
             var r_Wall = (Autodesk.Revit.DB.Wall)RevitWall.InternalElement;
 
             // wrapped exception catching to deliver more meaningful message in Dynamo
-            try 
+            try
             {
                 return Create.WallsFromRevitWall(r_Wall, DocumentManager.Instance.CurrentDBDocument);
             }
-            catch (Exception ex) {
+            catch (Exception ex)
+            {
                 throw new Exception(ex.Message);
             }
         }
     }
 
-    public static class Floor {
-        public static Elements.Floor[] FromRevitFloor(this Revit.Elements.Floor RevitFloor) {
+    public static class Floor
+    {
+        /// <summary>
+        /// Convert Revit walls to Elements.Floors for use in Hypar models
+        /// </summary>
+        /// <param name="RevitFloor">The floors to be exported</param>
```

Comment:
This is a single floor correct?

Reply:
yes, it is one floor, lacing handled by dynamo.  I've removed the "s"